index.html

```diff
@@ -1,60 +1,113 @@
 <!doctype html>
 <html>
-  <head>
-    <title>Wayback Machine</title>
-    <link rel="stylesheet" type="text/css" href="css/bootstrap.min.css">
-  </head>
-  <body style="min-width: 200px;max-width: 200px;">
-  	<table width="100%" style="min-width: 120px; ">
-      <tr class="spaceUnder">
-        <td>
-            <div style="margin: 0px; padding: 0px; border: 0px solid rgb(6, 117, 211); font-size: 19px; font-style: inherit; font-variant: inherit; font-weight: inherit; font-stretch: inherit; line-height: 1; font-family: sans-serif; vertical-align: baseline; box-sizing: content-box; overflow: unset; height: 1px; position: relative; width: auto; display: flex; background-color: rgb(9, 150, 248); color: rgb(255, 255, 255); text-align: left; align-items: center; border-radius: 3px; justify-content: center; text-decoration: none;"></div>
-                <div class='container'>
-                    <img class='resize_fit_center'
-                      src='images/logo.gif' />
-                </div>
-            <div style="margin: 0px; padding: 0px; border: 0px solid rgb(6, 117, 211); font-size: 19px; font-style: inherit; font-variant: inherit; font-weight: inherit; font-stretch: inherit; line-height: 1; font-family: sans-serif; vertical-align: baseline; box-sizing: content-box; overflow: unset; height: 1px; position: relative; width: auto; display: flex; background-color: rgb(9, 150, 248); color: rgb(255, 255, 255); text-align: left; align-items: center; border-radius: 3px; justify-content: center; text-decoration: none;"></div>
+<head>
+  <title>Wayback Machine</title>
+  <link rel="stylesheet" type="text/css" href="css/bootstrap.min.css">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
@rgpt thanks for taking the time to go through my code and reviewing it. As for your review, I know that CDN shouldn't be used in our extension. But, this PR is just an example of social sharing so that maintainers of this repo can get the idea. Many more options and tweaks are needed to make it a permanent enhancement in extension. So for the current purpose, CDN suffices the requirement.